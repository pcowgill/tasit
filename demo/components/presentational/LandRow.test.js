import React from "react";
import { shallow } from "enzyme";
import LandRow from "./LandRow";

describe("LandRow", () => {
  it("renders the component", async () => {
    expect(shallow(<LandRow land={{}} onPress={() => {}} />)).toMatchSnapshot();
  });
});
