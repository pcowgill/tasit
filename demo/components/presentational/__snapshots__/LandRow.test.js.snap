// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandRow renders the component 1`] = `
<TouchableHighlight
  activeOpacity={0.85}
  delayPressOut={100}
  onPress={[Function]}
  underlayColor="black"
>
  <Component
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "marginBottom": 13.34,
        "paddingBottom": 40.019999999999996,
        "paddingLeft": 22.5,
        "paddingRight": 22.5,
        "paddingTop": 40.019999999999996,
      }
    }
  >
    <Land
      land={Object {}}
    />
  </Component>
</TouchableHighlight>
`;
